etna/packages/etna-js/hooks/useFeatureFlag.js

```diff
@@ -0,0 +1,11 @@
+import {useState} from 'react'
+import {useReduxState} from "./useReduxState";
+
+export function useFeatureFlag(flagName) {
+  const user = useReduxState(({ user }) => user);
+
+  return useState(() => {
+    if (user && user.flags && user.flags[flagName]) return true;
```

Comment:
When we want janus tokens to support feature flags, we can just parse them into this attribute of the user object.